src/Sulu/Bundle/ContentBundle/Resources/public/js/components/smart-content/main.js

```diff
@@ -580,6 +591,7 @@ define(['services/husky/util'], function(util) {
                     this.sandbox.dom.append(ul, _.template(templates.contentItem)({
                         dataId: item[this.options.idKey],
                         value: item[this.options.titleKey],
+                        image: item[this.options.imageKey] ? item[this.options.imageKey] : null,
```

Comment:
no i thought that an empty string was returned ... but i am not sure i will change this to `item[this.options.imageKey] || null`